app/_src/kubernetes-ingress-controller/guides/services/custom-entity.md

```diff
@@ -43,7 +45,9 @@ This example configures custom entities for the `degraphql` plugin, which allows
 
 ## Create a GraphQL Service
 
-The `degraphql` plugin requires an upstream GraphQL API. For this tutorial, we'll use [Hasura] to create an example GraphQL service:
+The `degraphql` plugin requires an upstream GraphQL API. For this tutorial, we'll use [hasura] to create an example GraphQL service:
+
+[hasura]: https://hasura.io/
```

Comment:
```suggestion
The `degraphql` plugin requires an upstream GraphQL API. For this tutorial, we'll use [Hasura] to create an example GraphQL service:

[Hasura]: https://hasura.io/
```
Why switch this to lowercase? Based on a quick google, their product name is "Hasura" with a capital.

Reply:
Good call!